pywbem_mock/_mainprovider.py

```diff
@@ -71,7 +71,7 @@
 # pylint: disable=invalid-name
 
 
-class MainProvider(ResolverMixin, BaseProvider):
+class MainProvider(BaseProvider, ResolverMixin):
```

Comment:
Good catch

Reply:
Was needed as part of adding the init arguments in MainProvider to get super() to call BaseProvider first.